experimental/ptg/ptg.nan-latn-tw.taigipoj/source/ptg.nan-latn-tw.taigipoj.model.kps

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package>
+  <System>
+    <KeymanDeveloperVersion>13.0.101.0</KeymanDeveloperVersion>
+    <FileVersion>12.0</FileVersion>
+  </System>
+  <Options>
+    <ExecuteProgram></ExecuteProgram>
+    <ReadMeFile>readme.htm</ReadMeFile>
+    <MSIFileName></MSIFileName>
+    <MSIOptions></MSIOptions>
+  </Options>
+  <StartMenu>
+    <Folder></Folder>
+    <Items/>
+  </StartMenu>
+  <Info>
+    <Name URL="">PhahTaigi POJ Keyman Keyboard Lexical Model</Name>
```

Comment:
The lexical models are separate from Keyman keyboards.

In Keyman Developer, Package Editor, I suggest you change all the name references from `PhahTaigi POJ Keyman Keyboard Lexical Model` to `PhahTaigi POJ Lexical Model`